Cargo.toml

```diff
@@ -57,6 +57,7 @@ wasm = ["wasm-bindgen"]
 dump_lookahead_data = ["byteorder", "image"]
 
 [dependencies]
+histogram = "*"
```

Comment:
This needs to be set to an actual version (at least a semver-compatible major version). The reason is that crates can't be published to crates.io if they depend on a * version.